utils.js

```diff
@@ -27,9 +27,23 @@ export function getBuildPath(s) {
     const fl = s.split("Debug")[0];
     return outputPath + fl + "/debug/";
   }
+
+  if (s.toLowerCase() === "release") {
```

Comment:
adding release flavor would also require providing signing keys in the build step. I think it's better to take it as a separate PR.